modules/yarn/Makefile

```diff
@@ -0,0 +1,27 @@
+YARN := $(shell which yarn 2>/dev/null)
+
+export YARN_DIR?=frontend
+
+## Build frontend files using yarn
+yarn/build: $(YARN)
+	$(call assert-set,YARN)
+	set -ex \
+		&& ( \
+			cd $(YARN_DIR) \
+			&& $(YARN) --pure-lockfile --no-progress \
+			&& $(YARN) run build:prod \
+			&& $(YARN) autoclean \
+		)
+.PHONY: yarn/lint
+
+## Check frontend files using yarn
+yarn/lint: $(YARN)
+	$(call assert-set,YARN)
+	cd $(YARN_DIR) && $(YARN) lint
+.PHONY: yarn/lint
+
+## Fix frontend files using yarn
+yarn/fmt: $(YARN)
+	$(call assert-set,YARN)
+	cd $(YARN_DIR) && $(YARN) fmt:js
```

Comment:
I don't know if there are any tools to do this, but could we add fmt:css, too?
Or, should we just call fmt and leave it up to the developer to fmt both js and css?

Reply:
There are no targets in sherpa for fmt atm. When you guys come up with a strategy, let's talk then.

Reply:
By targets, I mean yarn scripts